packages/editor/src/accessibility/di.config.ts

```diff
@@ -1,7 +1,7 @@
+import type { BindingContext } from '@eclipse-glsp/client';
```

Comment:
@ivy-lli  note that on master lint does not complain about type only imports

Reply:
just also noticed this... strange.

Reply:
I think this is because BindingContext is a class and a class is in typescript either a type and code.
But for this type of detail the eslint ts roules need to be typed...

Reply:
the other thing should be already handled by typescript via the verbatimModule setting (or how this is called)